### PR TITLE
feat: add integ test workflow

### PR DESCRIPTION
**Pull request by sqin2019**
last part of #27 

Checked pantheon, binding was added and then removed.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -42,3 +49,43 @@ jobs:
...
+        uses: 'google-github-actions/auth@ef5d53e30bbcd8d0836f4288f5e50ff3e086997d' # ratchet:google-github-actions/auth@v1
+        with:
+          workload_identity_provider: '${{ env.WIF_PROVIDER }}'
+          service_account: '${{ env.WIF_SERVICE_ACCOUNT }}'
+          token_format: 'access_token'
+      - name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@e30db14379863a8c79331b04a9969f4c1e225e0b' # ratchet:google-github-actions/setup-gcloud@v1
+      - name: 'Set Condition Title'
+        run: |
+          run_id=${{ github.run_id }}
```
Comment (capri-xiyue) [inline .github/workflows/ci.yml:87]: Besides github.run_id, add `github.ref_name` to make devs easily differentiate between different PRs.

**Review by capri-xiyue — APPROVED** [review]

**Review by yolocs — APPROVED** [review]
